R/setup_package.R

```diff
@@ -1,4 +1,13 @@
 #' Add or update the checklist infrastructure to an existing package
+#'
+#' Use this function when you have an existing package and you want to use the
+#' checklist functionality.
+#' Please keep in mind that the checklist is an opinionated list of checks.
+#' It might require some breaking changes.
+#' DO READ `vignette("getting_started")` before you run this function.
```

Comment:
Same suggestion as before.
```suggestion
#' Please DO READ `vignette("getting_started")` before running this function.
```